app/models/github_repository.rb

```diff
@@ -58,6 +116,12 @@ def present?(**options)
     self.class.present?(@client, @id, options)
   end
 
+  def classroom_configs?(**options)
```

Comment:
With this I think they could have other types of configuration branches in the future. I do think something like `branch_present?` instead would be a good idea.